package testmessages;

message TestMessage {
    required int32 id = 1;
    required string txt = 2;
    optional WarningType wrn = 3;

    enum WarningType {
        UNSETTED = 0;
        CRIT = 1;
        NOT_CRIT = 2;
        SUPER_CRIT = 3;
    }
}

message TestMessageContainer {
    repeated TestMessage messages = 1;
}

message NetPackage {
    required int32 typeId = 1;
    required int32 time = 2;
    required string data = 3;
}